### PR TITLE
Embedded API endspoints and version variable

### PR DESCRIPTION
**Pull request by may55**
# Description

-Two variables added to the file opsdroid/const.py as "DIALOGFLOW_API_ENDPOINT" and  "DIALOGFLOW_API_VERSION" 
- Fix embedded these variable in opsdroid/parsers/dialogflow.py also.

**Fixes** #380 

## Status
**READY** 

# Checklist:

- [x] I have performed a self-review of my own code

### CODE REVIEW

**Comment by jacobtomlinson** [comment]
This is great many thanks! You're missing a space after the comma in your import line, that's why the tests are failing. If you can change that I'll get it merged.

File: opsdroid/parsers/dialogflow.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 import aiohttp
 
-from opsdroid.const import DEFAULT_LANGUAGE
+from opsdroid.const import DEFAULT_LANGUAGE, DIALOGFLOW_API_ENDPOINT, DIALOGFLOW_API_VERSION
```
Comment (jacobtomlinson) [inline opsdroid/parsers/dialogflow.py]: This line is also too long. Please break over multiple lines.

**Comment by may55** [comment]
Can you please help me how to proceed now in issue #380

File: opsdroid/parsers/dialogflow.py (hunk 1 above)
Comment (may55) [reply opsdroid/parsers/dialogflow.py]: Okay I am doing same.

**Comment by may55** [comment]
All checks have passed, what else should I need to change?

**Comment by may55** [comment]
Please review the changes.

**Comment by jacobtomlinson** [comment]
Hi really sorry for the delay I've been on holiday. Looks good to go!

**Comment by jacobtomlinson** [comment]
Ping me your address via DM on [Twitter](https://twitter.com/_jacobtomlinson) or [Gitter](https://gitter.im/opsdroid) and I'll post you some [opsdroid stickers](https://medium.com/opsdroid/stickers-for-contributors-a0a1f9c30ec1)!

**Pull request merged**